### ISSUE TITLE
fine-tuning config on COCO

### ISSUE THREAD

**Issue by TobiasLee** [microsoft/GenerativeImage2Text#41]
Thanks for your awesome work. 
Would you like to share the fine-tuning setup such as the number of training epochs, learning rate and optimizer configs, as I did not find corresponding in your paper.

**Comment by amsword** [comment]
I assume you are looking at the captioning task, and here is the parameters, shown in the suppl. of the paper.

<img width="1030" alt="image" src="https://user-images.githubusercontent.com/3356680/213548927-01b41c34-40c1-47a6-b892-9732f177b088.png">

**Comment by amsword** [comment]
feel free to pose any questions you have.

**Comment by TobiasLee** [comment]
Thanks for sharing the details. Did you use any regularization methods such as dropout  & weight decay?  Seems that there is no dropout used in the pre-training?

**Comment by amsword** [comment]
weight decay is 0.01; dropout is 0.1. I have not tried to completely remove the dropout, but I think the result should be similar.
